src/commands/dockerInstaller.ts

```diff
@@ -84,3 +79,26 @@ export class MacDockerInstaller extends DockerInstallerBase {
         terminal.show();
     }
 }
+
+export async function showDockerInstallNotification(): Promise<void> {
+    const dockerInstallMessage = ext.os.platform === 'linux'
+        ? localize('vscode-docker.commands.dockerInstaller.installDockerInfo', 'Docker is not installed. Would you like to open a page with more information?')
+        : localize('vscode-docker.commands.dockerInstaller.installDocker', 'Docker Desktop is not installed. Would you like to Install?');
+
+    const confirmationPrompt: vscode.MessageItem = ext.os.platform === 'linux' ? { title: 'Open' } : { title: 'Install' };
```

Comment:
Use the `learnMoreLink` option for Linux, instead of 'Open'. Also, use `ext.ui.showWarningMessage(...)`. This will show a 'Learn More' button which is more consistent with our existing style.

Reply:
Probably should change the text on the Linux prompt as well to fit that 'Learn More' button a little better.

Reply:
Not sure I follow your suggestion. Are you suggesting to use `ext.ui.showWarningMessage(...)` which will change the button from 'Open' to 'Learn More'? So effectively changing the label from 'open' to 'learn more' right?

Reply:
In the options object, `ext.ui.showWarningMessage(...)` has a `learnMoreLink` property, which you can use instead of the `Open` button. This would be more consistent with the rest of our UI which uses the `learnMoreLink` for similar things.

Reply:
Discussed offline, here we should stick with `window.showInformationMessage(...)`, and just change the text on that Open button (and probably the dialog as well) to `Learn more`. That way, the toast doesn't stay open when you click `Learn more`.

Reply:
Changed the button from 'Open' to 'Learn more'